nixos/doc/manual/configuration/x-windows.xml

```diff
@@ -96,6 +96,44 @@
   The options are named identically for all other display managers.
   </para>
  </simplesect>
+ <simplesect xml:id="sec-x11--graphics-cards-intel">
+  <title>Intel graphics drivers</title>
+  <para>
+   There are two choices for Intel graphics driver in X.org:
+   <literal>intel</literal> (provided by the package <package>xf86-video-intel</package>)
+   and <literal>modesetting</literal> (included in the <package>xorg-server</package> itself).
+   The default is <literal>modesetting</literal>.
+  </para>
+  <para>
+   The first driver is specific to Intel GPUs but is not recommended by most distribution:
+   it lacks several modern features (Glamor, 2D graphics acceleration via OpenGL, is not
+   possible with the latest DRI 3). The package was also last updated in 2015.
+   The second is a generic driver which uses the kernel
```

Comment:
```suggestion
   The second is a generic driver which uses the kernel
```

We should probably flip the two paragraphs - Start with the recommended driver, list the alternative second.

I'd personally also not number them ( "The first…The second"), but just call them by name.